src/tests/ftest/mpiio/llnl_mpi4py.py

```diff
@@ -72,4 +49,4 @@ def test_mpi4py(self):
         :avocado: tags=mpiio,mpich,mpi4py
         :avocado: tags=LlnlMpi4py,test_mpi4py
         """
-        self.run_test(self.get_test_repo('mpi4py'), "mpi4py")
+        self.run_test(self.params.get('mpi4py', '/run/test_repo/'), "mpi4py")
```

Comment:
We need a wildcard character added here:
```suggestion
        self.run_test(self.params.get('mpi4py', '/run/test_repo/*'), "mpi4py")
```
So that https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-14163/13/artifact/Functional%20on%20EL%208.8/mpiio/llnl_mpi4py.py/job.log will get the value from the test yaml and not use the default:
```
2024-05-06 18:28:48,597 parameters       L0143 DEBUG| PARAMS (key=llnl, path=/run/test_repo/, default=None) => 'testmpio'
```